docs/modules/deploy/pages/deploying-in-kubernetes.adoc

```diff
@@ -30,7 +30,9 @@ https://helm.sh/[Helm^] is a package manager for Kubernetes. Hazelcast is distri
 See the https://github.com/hazelcast/charts[Hazelcast Helm Charts documentation^]
 for more details.
 
-== Kubernetes/OpenShift Operator
+== Helm-based Kubernetes/OpenShift Operators
+
+NOTE: The following section is related to Hazelcast Operators which are generated from Helm Charts and therefore their functionality is limited to what Helm offers. For the Full-Lifecycle Autonomous Operartor please check xref:deploy:deploying-with-operator.adoc[].
```

Comment:
Rather than introducing the Enterprise Operator here, it would be better to introduce it as a separate option.

== Enterprise Operator

Description of what the Enterprise Operator is, how it differs from the Helm-based Operators, and link to the tutorial.

Reply:
There's also a type in 'Operartor'

Reply:
Yeah, good idea. Updated.